### ISSUE TITLE
Cannot import name inputstream

### ISSUE THREAD

**Issue by diegopasti** [xhtml2pdf/django-xhtml2pdf#19]
Good evening,
Used django-xhtml2pdf in my application, until a few weeks ago on a windows machine 7. Unfortunately my machine spoiled and had to buy another that came with Windows 10, all the rest has been operating normally except the part print PDF. The error has shown is this:

Django Version: 1.9.7
Exception Type: ImportError
**Exception Value: Can not import name inputstream**

Exception Location: C: \ python27 \ lib \ site-packages \ xhtml2pdf \ parser.py in <module>, line 17
Python Executable: C: \ python27 \ python.exe
Python Version: 2.7.12

The line where the error occurs is in this file: C: \ python27 \ lib \ site-packages \ xhtml2pdf \ parser.py in <module>

In this line:
**from html5lib import treebuilders, inputstream**

I thought it could be some incompatibility with Windows 10 but I tested the project in a Linux environment and the same error occurs in exactly the same way.

Unfortunately google only helped me translating this text, almost did not think anything about something relevant to help solve this problem. I do not know if that might be the problem and apologize here not be the right place to share it. Being a bug or not is the record of someone who went through this problem and that seeking a solution ..

I think I may have to do with changing versions of a component installed with xthml2pdf but not sure ..

Once again I apologize for the inconvenience and thank you from of attention.

**Comment by diegopasti** [comment]
I found the problem! django-xhtml2pdf use xhtml2pdf which in turn used the html5lib .. this has gone through an upgrade that does not work perfectly with the latest version of xhtml2pdf. When xhtml2pdf through their requirements lowered the html5lib lowered the newest version which recently discovered this incompatibility.
As the project xhtml2pdf was officially discontinued on 05/02/2016 and it to ca I did not need to reinstall the project when needed to reinstall the problem appeared.

The solution and further details are described in a case reported in the project page (https://github.com/xhtml2pdf/xhtml2pdf/issues/318), just install the version 1.0b8.

**pip uninstall html5lib**
**pip install html5lib==1.0b8**

If anyone has any more trouble as I had, try installing xhtml2pdf before django-xhtml2pdf, and thus managed to solve my problem ..

Thanks anyway .. and is a tip for those who can most interesting ..;)
